### PR TITLE
EPMRPP-37007 UI: Events monitoring. Layout

### PR DESCRIPTION
**Pull request by andrei-bulagin**

### CODE REVIEW

File: app/src/pages/admin/projectsPage/img/project-events-inline.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/img/project-events-inline.svg]: Please move all icons to `common/img`

File: app/src/pages/admin/projectsPage/projectsPage.jsx [hunk 2]
```diff
@@ -1 +1,175 @@
...
+    icon: ProjectEventsIcon,
+  },
+];
+
+@connect(
+  (state) => ({
+    projectId: projectIdSelector(state),
+    section: projectSectionSelector(state),
+  }),
+  (dispatch) => ({
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/projectsPage.jsx]:
```
{
redirectToSection: (projectId, section) => rfrRedirect({
    type: PROJECT_DETAILS_PAGE,
    payload: { projectId, projectSection: section },
  })
}
```

File: app/src/pages/admin/projectsPage/projectsPage.jsx [hunk 3]
```diff
@@ -1 +1,175 @@
-export const ProjectsPage = () => <h1>Projects page</h1>;
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { redirect as rfrRedirect } from 'redux-first-router';
```
Comment (DiscoElevator) [inline app/src/pages/admin/projectsPage/projectsPage.jsx]: is this rename necessary?

File: app/src/pages/admin/projectsPage/projectsPage.jsx [hunk 4]
```diff
@@ -1 +1,175 @@
...
+  },
+  eventsTitle: {
+    id: 'ProjectDetailsPageEvents.title',
+    defaultMessage: 'Events',
+  },
+});
+
+const HEADER_BUTTONS = [
+  {
+    key: 'members',
```
Comment (AmsterGet) [inline app/src/pages/admin/projectsPage/projectsPage.jsx]: May be we need to create `projectDetailsSections` in `common/constants` and hold these values here

File: app/src/routes/routesMap.js [hunk 5]
```diff
@@ -78,6 +79,8 @@ export default {
   API_PAGE: '/api',
 
   [PROJECTS_PAGE]: '/administrate/projects',
+  [PROJECT_DETAILS_PAGE]:
+    '/administrate/project-details/:projectId/:projectSection(settings|members|events)?',
```
Comment (DiscoElevator) [inline app/src/routes/routesMap.js]: We discussed this route with Ilya, and agreed on:
`/administrate/:projectId/:projectSection(settings|members|events)?`

File: app/src/pages/admin/projectsPage/projectsPage.jsx [hunk 6]
```diff
@@ -1 +1,175 @@
...
+    return (
+      <div className={cx('header-buttons')}>
+        {HEADER_BUTTONS.map(({ key, text, icon }) => (
+          <GhostButton
+            key={key}
+            disabled={section === key}
+            icon={icon}
+            onClick={this.onHeaderButtonClick(key)}
+          >
+            {text}
```
Comment (AmsterGet) [inline app/src/pages/admin/projectsPage/projectsPage.jsx]: Should we use `formatMessage` here?

File: app/src/routes/routesMap.js (hunk 5 above)
Comment (AmsterGet) [inline app/src/routes/routesMap.js]: May be better to define the next path: `/administrate/projects/:projectId/:projectSection(settings|members|events)?`
to be more apparent (as extend `PROJECTS_PAGE`)

File: app/src/pages/admin/projectsPage/img/project-events-inline.svg (hunk 1 above)
Comment (andrei-bulagin) [reply app/src/pages/admin/projectsPage/img/project-events-inline.svg]: Moved

File: app/src/pages/admin/projectsPage/projectsPage.jsx (hunk 2 above)
Comment (andrei-bulagin) [reply app/src/pages/admin/projectsPage/projectsPage.jsx]: Done

File: app/src/pages/admin/projectsPage/projectsPage.jsx (hunk 3 above)
Comment (andrei-bulagin) [reply app/src/pages/admin/projectsPage/projectsPage.jsx]: Not really

File: app/src/routes/routesMap.js (hunk 5 above)
Comment (andrei-bulagin) [reply app/src/routes/routesMap.js]: Changed

File: app/src/pages/admin/projectsPage/projectsPage.jsx (hunk 4 above)
Comment (andrei-bulagin) [reply app/src/pages/admin/projectsPage/projectsPage.jsx]: Done

File: app/src/pages/admin/projectsPage/projectsPage.jsx (hunk 6 above)
Comment (andrei-bulagin) [reply app/src/pages/admin/projectsPage/projectsPage.jsx]: Fixed
